modules/package.json

```diff
@@ -1,5 +1,6 @@
 {
 	"devDependencies": {
+		"@liferay/eslint-config": "25.0.1",
```

Comment:
Do we really need to also specify it as a `devDependency`? I would've expected that since it's coming anyways through `@liferay/npm-scripts` that the resolution would take care of that and provide the right version either way.

Reply:
I think we have to specify it since it is a different version than exists in DXP. If we had multiple versions brought in through other modules, we wouldn't have to.

Reply:
^ I am second guessing myself here actually

Reply:
You are right, I can remove that. Even if we bump the version in resolutions, yarn will install the bumped version